api/docs/release.dox

```diff
@@ -173,6 +173,10 @@ Further non-compatibility-affecting changes include:
  - Added opnd_create_immed_double(), opnd_get_immed_double() and
    opnd_is_immed_double() to enable the creation and handling of double
    precision floating-point operands.
+ - Added -reset_at_created_thread_count option to allow triggering reset when
```

Comment:
Please also add to api/docs/bt.dox which has a section on adaptive instrumentation where a statement about the callback being the way to do phased instrumentation in a race-free manner would be good.  (Search for dr_flush_region).

Reply:
Done.